### PR TITLE
Skeleton for connecting to postgres and .env loader.

### PR DESCRIPTION
**Pull request by aria-afk**
Basic sub-package to create a connection to postgres. Also added a utils package since I think we will need to share some code pieces! 

Feel free to ask questions and or point out pieces you don't like!

#### Testing

(from /go-get-tickets)

To test utils: `go test ./utils`

to test pg: `go test ./pg`

### CODE REVIEW

File: utils/env.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// utils/env.go: Tools to load or manage .env files
+package utils
+
+import (
+	"github.com/joho/godotenv"
+)
+
+// Load a specified .env file by specified path (relative)
+func LoadEnv(filePath string) error {
```
Comment (aria-afk) [inline utils/env.go:9]: I guess this is a bit redundant if we just want to import `godotenv` anywhere we need it but I do kind of like the idea of having a utils import for a bunch of stuff eventually.

Comment (cjdesno) [reply utils/env.go:9]: yeah i think that could be useful

File: pg/pg_test.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import (
+	"os"
+	"testing"
+
+	"github.com/aria-afk/go-get-tickets/utils"
+)
+
+func setup() {
+	os.WriteFile("pg_test.env", []byte("PG_CONN_STRING=postgresql://postgres:test@localhost/template1"), 0755)
```
Comment (cjdesno) [inline pg/pg_test.go:11]: will this write the file in the root directory? might be nice to put it in a test directory that we can gitignore or something

Comment (aria-afk) [reply pg/pg_test.go:11]: They will get added to the modules dir (so in this case `pg/`) also, oops for all the file stuff we can just do `defer ...` to ensure nothing gets left around

**Review by cjdesno — APPROVED** [review]

**Pull request merged**